src/services/index.js

```diff
@@ -34,10 +48,14 @@ export const fetchGenres = async () => {
       console.log('json ERROR: ', error);
     });
   console.log(result);
-  localStorage.setItem('genresList', JSON.stringify(result.genres));
+  setInStorage('genresList', result.genres);
   return (result.genres);
 };
 
-fetchGenres();
+export const filterByReference = (arr1, arr2) => {
```

Comment:
this function can be simplified in one line and by using `includes` array method